python/ray/util/metrics.py

```diff
@@ -149,6 +150,43 @@ def __reduce__(self):
         serialized_data = (self._name, self._description, self._tag_keys)
```

Comment:
L149 should the deserializer be renamed to `Counter` too? I'm getting a lot of warnings when pickling the Counter objects to remote tasks:

```
WARNING metrics.py:206 -- `metrics.Count` has been renamed to `metrics.Counter`. `metrics.Count` will be removed in a future release.
```

Reply:
Oh, good catch @franklsf95! Any chance you would like to make a PR fixing it?